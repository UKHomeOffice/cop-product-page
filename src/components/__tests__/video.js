import React from "react";
import renderer from "react-test-renderer";
import Video from "../video";

describe("video", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Video/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct width, height, source with its correct values given", () => {
    const tree = renderer
      .create(<Video width={'20'} height={'10'} source={'http://test.com'}/>);
    const testInstance = tree.root;
    expect(testInstance.props).toHaveProperty('width', '20');
    expect(testInstance.props).toHaveProperty('height', '10');
    expect(testInstance.props).toHaveProperty('source', 'http://test.com');
  });
});