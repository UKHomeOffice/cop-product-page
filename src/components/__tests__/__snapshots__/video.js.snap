// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`video renders correctly 1`] = `
<div
  className="responsive-video"
>
  <video
    controls={true}
    playsInline={true}
  >
    <source
      type="video/mp4"
    />
  </video>
</div>
`;
